src/util/tags.ts

```diff
@@ -36,6 +36,8 @@ function tagify(
     input: HTMLInputElement | null,
     { whitelist, maxTags, enforceWhitelist }: TagifyOptions = {}
 ): Tagify<TagRecord> | null {
+    enforceWhitelist ??= true;
```

Comment:
hiiigher

Reply:
Funny enough I independently noticed what you meant. I think I wasn't aware you could set defaults for objects destructured in the params.